packages/deployments/contracts/contracts/lib/Connext/ConnextLogic.sol

```diff
@@ -714,6 +718,11 @@ library ConnextLogic {
     if (_args.xCallArgs.params.to == address(0)) {
       revert ConnextLogic__xcall_emptyTo();
     }
+
+    // ensure callback fee is zero when callback address is not contract
+    if (!AddressUpgradeable.isContract(_args.xCallArgs.params.callback) && _args.xCallArgs.params.callbackFee > 0) {
+      revert ConnextLogic__xcall_nonZeroCallbackFeeForNonContractCallback();
+    }
```

Comment:
we should allow 0 fees as this will happen in an hour and if a relayer chooses to forward without a fee, that is their decision to make.

Reply:
Okay